### PR TITLE
Feature/95 enhance travis theme

### PR DESCRIPTION
**Pull request by eddiejaoude**
screenshot 2015-03-17 09 31 56

### CODE REVIEW

File: src/DashboardHub/Bundle/AppBundle/Service/TravisService.php [hunk 1]
```diff
@@ -47,9 +48,34 @@ public function getBuilds($reponame, $max = 5)
                 ,
                 true
             );
-            $cache->set($builds, 600);
+            $cache->set($builds, 60);
```
Comment (brettminnie) [inline src/DashboardHub/Bundle/AppBundle/Service/TravisService.php]: Should this not be a config value?

Comment (eddiejaoude) [inline src/DashboardHub/Bundle/AppBundle/Service/TravisService.php]: Yep, I will sort it.

Comment (brettminnie) [inline src/DashboardHub/Bundle/AppBundle/Service/TravisService.php]: In branch or as a new pr?

On Tue, Mar 17, 2015 at 4:38 PM, Eddie Abou-Jaoude <notifications@github.com

Comment (eddiejaoude) [inline src/DashboardHub/Bundle/AppBundle/Service/TravisService.php]: I was going to do it in this branch, but you know what, lets get it out there & I will create a Ticket to do it & assign it to me.

Comment (eddiejaoude) [inline src/DashboardHub/Bundle/AppBundle/Service/TravisService.php]: Created https://github.com/DashboardHub/PipelineDashboard/issues/100

**Comment by brettminnie** [comment]
Merged #100 noted as improvement

**Pull request merged**